### ISSUE TITLE
Combine interactive and doctor mode

### ISSUE THREAD

**Issue by jerone** [raineorshine/npm-check-updates#1224]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 14.14`

---

Would it be possible to combine interactive and doctor mode. 
After choosing which upgrades to make, I would like to run the tests, and restore if tests fail.

Currently, it completely ignores interactive mode:
```
$ ncu --interactive --doctor
Running tests before upgrading
npm install
npm run test
Upgrading all dependencies and re-running tests
ncu --interactive
npm install
npm run test
✓ Tests pass
```

**Comment by raineorshine** [comment]
Yes, thanks for asking. I enabled `--interactive` with `--doctor`. I also added the full test logs to stdout. They were being silenced before, which not only made it impossible to see the progress of the tests, but suppressed the error messages when the tests failed.

Published to `v16.4.3`. Please give it a try at your convenience.

Note that the tests are run once before updates are checked in order to ensure a working starting point. This is done as a safety precaution as doctor can report false negatives if the tests are failing before any upgrades are made. This makes for a slightly awkward user experience in that you have to wait for the tests to complete before you can choose the upgrades, but it ensures that the results can be relied on.

**Comment by jerone** [comment]
I just tried this on an success path and it's working great. Thanks.
